src/helpers/load.ts

```diff
@@ -251,3 +244,10 @@ function validateUrl(url?: string) {
 function warn(value: string, type: string, line: number) {
   console.warn(`Row ${line + 2}: “${value}” is not a valid ${type}.`);
 }
+
+// helper to replace curly quotes with straight ones
+export function sanitizeQuotes(str: string): string {
+  return str
+    .replace(/[‘’‛‚]/g, "'") // curly single quotes
+    .replace(/[“”„‟]/g, '"'); // curly double quotes
+}
```

Comment:
don't need this version now- i'd remove it and import the one from stringUtils. otherwise lgtm!